src/components/Loader/Loader.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import Loader from './index';
+
+describe('Loader component', () => {
+  test('renders with testId Loader', () => {
+    render(<Loader />);
+    const loader = screen.getByTestId('Loader');
+    expect(loader).toBeTruthy();
+  });
+
+  test('renders with default className', () => {
+    render(<Loader />);
+    const loader = screen.getByTestId('Loader');
+    expect(loader).toBeTruthy();
+    expect(loader).toHaveClass('Loader');
+  });
+
+  test('renders with role Button', () => {
+    render(<Loader />);
+    const loader = screen.getByRole('button');
+    expect(loader).toBeTruthy();
+  });
+
+  test('renders with className Test', () => {
+    render(<Loader className="Test" />);
+    const loader = screen.getByTestId('Loader');
+    expect(loader).toBeTruthy();
+    expect(loader).toHaveClass('Test');
+  });
+
+  test('renders with LoadingIcon', () => {
+    render(<Loader />);
+    const loader = screen.getByTestId('Loader');
+    expect(loader).toBeTruthy();
+
+    const icon = loader.querySelector('.mdi-icon');
+    expect(icon).toBeTruthy();
+    expect(icon?.getAttribute('width')).toEqual('15.35');
```

Comment:
Can switch to this so that you don't need to optional chain it:

`expect(icon.style).toHaveProperty('height', '15.35px');`

Reply:
hmmm, 
getting an error if i try the suggestion with `querySelector` - 
not sure if i can query `.mdi-icon` without affecting the `LoadingIcon`

```
  expect(received).toHaveProperty(path, value)

    Expected path: "height"

    Expected value: "15.35"
    Received value: ""
```

Reply:
I guess i can pass a data-testId - one second